src/components/SearchForm/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./style.css";
+
+const SearchForm = ({ getResult }) => {
+  const HandleSubmit = (e) => {
+    e.preventDefault();
+    const searchText = e.target.username.value;
+    getResult(searchText);
+    e.target.reset();
```

Comment:
Well... this is easier! I've seen this before but completely forgot about it
(referring to `.reset()`)